app/javascript/packages/document-capture/components/review-issues-step.jsx

```diff
@@ -6,6 +6,7 @@ import AcuantCapture from './acuant-capture';
 import SelfieCapture from './selfie-capture';
 import FormErrorMessage from './form-error-message';
 import ServiceProviderContext from '../context/service-provider';
+import './review-issues-step.scss';
```

Comment:
I didn't realize JS could import CSS files? 🤯

Reply:
> I didn't realize JS could import CSS files? 🤯

Not JS, but Webpack. It can feel like a bit of an ugly hack, but it's nice to keep things self-contained, and leaves open some options for omitting unused styles.

The main alternative here could be to have something like a manifest file or SASS entrypoint which then loads all of the individual stylesheets for the package.

As you noticed below, this is a feature Webpacker provides for free by providing an option to extract CSS imported this way.

Related:

- https://github.com/rails/webpacker/blob/master/docs/css.md#link-styles-from-your-rails-views
- https://github.com/rails/webpacker/blob/master/docs/css.md#importing-css-from-js